tests_and_analysis/test/euphonic_test/test_spectrum2dcollection.py

```diff
@@ -0,0 +1,214 @@
+"""Unit tests for Spectrum2DCollection"""
+
+# Stop the linter from complaining when pytest fixtures are used idiomatically
+# pylint: disable=redefined-outer-name
+
+from typing import Optional
+
+import numpy as np
+import pytest
+
+from euphonic import Quantity, ureg
+from euphonic.spectra import OneLineData, Spectrum2D, Spectrum2DCollection
+
+from tests_and_analysis.test.utils import get_data_path
+from .test_spectrum2d import check_spectrum2d, get_spectrum2d
+
+
+def get_spectrum2dcollection_path(*subpaths):
+    """Get Spectrum2DCollection reference data path"""
+    return get_data_path('spectrum2dcollection', *subpaths)
+
+
+def get_spectrum2dcollection(json_filename):
+    """Get Spectrum2DCollection reference data object"""
+    return Spectrum2DCollection.from_json_file(
+        get_spectrum2dcollection_path(json_filename))
+
+
+@pytest.fixture
+def quartz_fuzzy_collection() -> Spectrum2DCollection:
+    """Coarsely sampled quartz bands in a few directions"""
+    return get_spectrum2dcollection("quartz_fuzzy_map.json")
```

Comment:
Do these accept `pathlib.Path`? Have you tested it?

Reply:
Only `str` is promised/documented at the moment. But Path should work. I'll create an Issue to look at that.